config.go

```diff
@@ -17,3 +17,8 @@ func SetupStorage(stg data.Storage) {
 func SetupUser(username string) {
 	global.User = &data.CalUser{Name: username}
 }
+
+// SetupSupportedComponents sets all components which are supported by this storage implementation.
+func SetupSupportedComponents(components []string) {
+	global.SupportedComponents = components
+}
```

Comment:
Looks good. Can you please update the Readme with this new addition? Perhaps, after the `Storage & Resources` section, add a new section `Configuration` so that all the configuration options can be described there. Do it for this function, and then I'll add the other ones to there.

Reply:
Done.